keg_auth/libs/authenticators.py

```diff
@@ -108,6 +108,9 @@ def __call__(self, *args, **kwargs):
 
         return resp or self.render()
 
+    def head(self):
+        return flask.abort(405)
```

Comment:
@guruofgentoo Sorry I didn't notice this prior to it getting merged, but according to [spec](https://tools.ietf.org/html/rfc7231#section-6.5.5), you're supposed to include an "allow" header when returning a 405:
> The origin server MUST generate an Allow header field in a 405 response containing a list of the target resource's currently supported methods.

Reply:
Good point - I'll make a revision.